packages/plugins/integration-zapier-ui/src/lib/components/zapier-settings/zapier-settings.component.html

```
@@ -0,0 +1,48 @@
+				<div
+					*ngIf="form.get('webhookUrl').touched && form.get('webhookUrl').errors?.pattern"
+					class="error-message"
+				>
+					{{ 'FORM.VALIDATION.INVALID_URL' | translate }}
+				</div>
+			</div>
+			<div class="hint">
+				<nb-icon icon="info-outline"></nb-icon>
+				{{ 'INTEGRATIONS.ZAPIER_PAGE.SETTINGS.WEBHOOK_INFO' | translate }}
+			</div>
+			<button nbButton status="primary" [disabled]="form.invalid || loading" size="small" outline type="submit">
```

HTMLHint (1.5.0)

[error] The attribute name of [ nbButton ] must be in lowercase.

(attr-lowercase)